MeetupPlatformApi/Features/Meetups/GetMeetups/Filter/ConfigurationQuery/MeetupsFilterHelper.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.ConfigurationQuery;
+
+using MeetupPlatformApi.Domain;
+using MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.FilterSettings;
+using System.Linq;
+
+public static class MeetupsFilterHelper
+{
+    public static IQueryable<Meetup> GetMeetupsFilteredByFilterSettings(
+        IQueryable<Meetup> meetupsQuery,
+        MeetupsFilterSettings meetupFilterSettings)
+    {
+        meetupsQuery = meetupsQuery.Filter(meetupFilterSettings);
+        meetupsQuery = SortSelection(meetupsQuery, meetupFilterSettings.SortOption);
+        var skippedMeetups = (meetupFilterSettings.PageNumber - 1) * meetupFilterSettings.PageSize;
```

Comment:
I would be recommend you to create `private` method for get skipped meetups. We will adhere to 'Single responsibility' principle.